### PR TITLE
chore: add links to pkg.go.dev for function calls

### PR DESCRIPTION
**Pull request by jbrockopp**
Part of the effort for https://github.com/go-vela/community/issues/54

Adding links to "godoc" (https://pkg.go.dev/) for function calls in the code 👍

### CODE REVIEW

File: executor/local/build.go [hunk 1]
```diff
@@ -207,6 +215,8 @@ func (c *client) AssembleBuild(ctx context.Context) error {
 // nolint: funlen // ignore function length - will be refactored at a later date
```
[the remark this answers is not in the document]
Comment (jbrockopp) [reply executor/local/build.go]: Fixed 👍

**Review by KellyMerrick — APPROVED** [review]
lgtm

File: executor/linux/step.go [hunk 2]
```diff
@@ -91,13 +95,13 @@ func (c *client) PlanStep(ctx context.Context, ctn *pipeline.Container) error {
...
 
 	// update the step container environment
+	//
+	// https://pkg.go.dev/github.com/go-vela/pkg-executor/internal/step#Environment
 	err = step.Environment(ctn, c.build, c.repo, _step, c.Version)
 	if err != nil {
 		return err
 	}
 
-	_step.SetStatus(constants.StatusSuccess)
```
Comment (matt-fevold) [inline executor/linux/step.go:99]: was this an intentional removal?

**Review by matt-fevold — CHANGES_REQUESTED** [review]

File: executor/linux/step.go (hunk 2 above)
Comment (jbrockopp) [reply executor/linux/step.go:99]: Yeah that was carry out from some of the past refactoring efforts that I missed submitting in those PRs 😞

**Review by matt-fevold — APPROVED** [review]

**Pull request merged**